src/_experimental.scss

```diff
@@ -4,6 +4,37 @@
 	- Experimental  CSSspecific for view port are defined in the "views/[view port]" theme CSS
 */
 
+// Colours for the provisional alert styles
+// TODO: Rename and migrate these into _variables.scss if the provisional alert styles become official in the future
+$provisional-alert-danger-color: #d3080c;
+$provisional-alert-warning-color: #ee7100;
+$provisional-alert-info-color: #269abc;
+$provisional-alert-success-color: #278400;
```

Comment:
@duboisp Should I scrap these and just directly pass the HEX colours as parameters for the ``provisional-alert-type()`` mixin? These colour variables are no longer being used multiple times.

Reply:
it's up to you, but having those variable written here will ease the move when that provisional would be official.

Reply:
Ok, will keep them as-is.